src/python/pants/backend/python/rules/run_setup_py_test.py

```diff
@@ -161,7 +157,7 @@ def test_generate_chroot(self) -> None:
                 "name": "foo",
                 "version": "1.2.3",
                 "package_dir": {"": "src"},
-                "packages": ["foo", "foo.qux", "foo.resources.js"],
```

Comment:
This was a bug, right?

Reply:
I think so (see description of this PR).